src/api/controllers/comments.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable arrow-body-style */
+import models from '../../sequelize/models';
+
+const comments = {
```

Comment:
You're using objects and we're using classes on controllers. WHY?

Reply:
I did not know!